### ISSUE TITLE
Config: textarea without label

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#87]
> All interactive elements should have a name or label, otherwise a screen reader will not say what it is for. The DNSma[s]q field does not have a name or label to identi[f]y its purpose.

<img width="266" height="137" alt="Image" src="https://github.com/user-attachments/assets/8431c6b4-f3d2-4a79-820d-13166fcbe491" />

[4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
